src/item.cpp

```diff
@@ -9921,6 +9666,32 @@ const cata::value_ptr<islot_comestible> &item::get_comestible() const
     }
 }
 
+item &item::get_consumable_from( const Character &eater )
```

Comment:
looking at this function in a vacuumm, it doesn't really make sense. you can have multiple consumables, who's to say which one you want?